site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -192,6 +192,21 @@ CustomTabsIntent intent = new CustomTabsIntent.Builder()
     .setSecondaryToolbarViews(secondaryToolbarViews, clickableIds, toolbarPendingIntent)
     .build();
 ```
+## Add an action button to the toolbar
+
+To add an additional action button to the toolbar, use the intent extra `EXTRA_SHOW_ON_TOOLBAR`:
+
+`actionIntent.putExtra(ActionBroadcastReceiver.EXTRA_SHOW_ON_TOOLBAR, true);`
```

Comment:
Reading only this section I don't understand where the `actionIntent` is coming from. Also, how do I specify label and icon for the button?

Reply:
Updated the intent syntax from your example. I suspect this is supposed to be part of the button creation process that's already in the doc, but I don't know how they fit together.